importer/views.py

```
@@ -27,25 +31,54 @@
+    Reads from Setting.input_directory if available, otherwise falls back to
+    /input (if running in Docker) or ~/input.
+
+    Returns:
+        Path to the input directory as a string.
+    """
+    # Try to get the configured directory from settings
+    try:
+        setting = Setting.objects.first()
+        if setting and setting.input_directory:
+            return setting.input_directory
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)